src/main/java/com/puppycrawl/tools/checkstyle/checks/javadoc/JavadocMethodCheck.java

```diff
@@ -338,12 +338,7 @@ public void setAllowMissingReturnTag(boolean flag) {
 
     @Override
     public final int[] getRequiredTokens() {
-        return new int[] {
-            TokenTypes.CLASS_DEF,
-            TokenTypes.INTERFACE_DEF,
-            TokenTypes.ENUM_DEF,
-            TokenTypes.RECORD_DEF,
-        };
+        return CommonUtil.EMPTY_INT_ARRAY;
```

Comment:
Do we have a test case with no tokens? We should add one if not, and also generate a regression report with the same config.

Reply:
It depends on what you mean. BDD, even when using default, lists all the tokens in the input file. I don't know how it works under the hood if it supplies an empty `tokens`, the same list as the default, or what.

34 input files for this check use `tokens = (default)`. So we have inputs that verify the default tokens property.